src/Uno.UI/UI/Xaml/Controls/ContentDialog/ContentDialog.cs

```diff
@@ -260,9 +262,14 @@ void Complete(ContentDialogButtonClickEventArgs a)
 				if (!a.Cancel)
 				{
 					const ContentDialogResult result = ContentDialogResult.None;
-					_tcs.SetResult(result);
+
 					CloseButtonCommand.ExecuteIfPossible(CloseButtonCommandParameter);
-					Hide(result);
+
+					if (Hide(result))
+					{
+						Console.WriteLine("tcs close");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5229032)